styles/go.dev/catppuccin.user.css

```diff
@@ -0,0 +1,930 @@
+    }
+  }
+  @media (prefers-color-scheme: dark) {
+    :root[data-theme='auto'] {
+      #catppuccin(@darkFlavor, @accentColor);
+    }
+  }
+  :root[data-theme="dark"] {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+  :root[data-theme="light"] {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    background-color: @base;
+
+    --abbey: @mantle;
+    --black-1: @crust;
+    --color-border: @surface2;
+    --color-background: @base;
+    --color-background-accented: @mantle;
+    --color-background-banner: @mantle;
+    --color-background-card-footer: @crust;
+    --color-background-code: @surface0; // Code Snippets
+    --color-background-inverted: @crust;
+    --color-background-logo: @text;
+    --color-background-playground-input: @mantle;
+    --color-brand-primary: @accent-color; // Affects most Go-Color Elements
+    --color-button: @accent-color;
+    --color-button-accented: @accent-color;
+    --color-button-text: @crust;
+    --color-button-text-disabled: @subtext0;
+    --color-bright-text-link: @accent-color;
+    --color-text: @text;
+    --color-text-link: @accent-color;
+    --color-text-subtle: @subtext1;
+    --focus-box-shadow: 0 0 .0625rem .0625rem @accent-color; // Only modifies color;
+    --gray-2: @surface0;
+    --gray-3: @text;
+    --gray-6: @crust;
+    --pink: @red;
+    --white: @text;
+
+    .btn {
+      background: @accent-color;
+      color: @crust;
+    }
+    // /dl
```

Comment:
Does this comment need to be here? Not really sure what it is trying to say lol.